src/components/Resizer/Resizer.jsx

```diff
@@ -12,26 +12,36 @@ const {
 } = React.PropTypes;
 
 /**
- * {"categories": ["utility"]}
- *
- * This is a helper component used for getting the width and height of a
- * containing element. This component doesn't take normal children. It expects
- * you to pass a single function for children. It will then call that function
- * with new `width` and `height` values if the container size changes.
+ * Contains the base functionality of `Resizer` but allows for injection of
+ * `listenTo` and `removeListener` function dependencies to improve
+ * testability.
  */
-const Resizer = createClass({
-	displayName: 'Resizer',
+export const ResizerBase = createClass({
```

Comment:
I added this base component just to make it more convenient to mock the dependencies of `Resizer`. It provides a way of injecting the dependencies for `Resizer` to obviate the need for using Rewire to change internal values.